src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -24,10 +24,28 @@
 
 /**
  * @author Andreas Möller <am@localheinz.com>
+ * @author SpacePossum
  */
 final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
     /**
+     * @internal
+     */
+    const SET_INTERNAL = '@internal';
+
+    /**
+     * @internal
+     */
+    const SET_ALL = '@all';
+
+    /**
+     * Subset of SET_INTERNAL.
+     *
+     * @internal
+     */
+    const SET_COMPILER_OPT = '@compiler_opt';
```

Comment:
Any reason to use `opt` instead of `optimized`?

Reply:
not really, it is kind of a common abbreviation I think?

Reply:
i got opt as optional :D

Reply:
renamed